src/bfx_tools/sambamba.ml

```diff
@@ -0,0 +1,36 @@
+open Biokepi_run_environment
+open Common
+
+module Remove = Workflow_utilities.Remove
+
+
+(* Filter language syntax at
+   https://github.com/lomereiter/sambamba/wiki/%5Bsambamba-view%5D-Filter-expression-syntax *)
+let view ~(run_with : Machine.t) ~bam ~filter output_file_path =
```

Comment:
SGTM, was thinking about it but thought that'd be an easy addition if the day comes.